styles.css

```diff
@@ -78,7 +82,189 @@
   margin: 58px 0;
 }
 
+/*
+According to linters this should be placed here
+*/
+.aboutme-card img {
+  height: 80px;
+  width: 80px;
+  margin: 40px auto;
+}
+
 #icons-div img {
   padding: 1rem;
   background-color: transparent;
 }
+
+/*
+Feature 2 Styles Section
+*/
+#arrow-btn {
+  padding: 8px;
+}
+
+.works-section {
+  display: grid;
+  grid-template-rows: 8;
+  text-align: center;
+  row-gap: 24px;
+  justify-items: center;
+}
+
+.headings-2 {
+  font-family: 'Inter', 'Montserrat', sans-serif;
```

Comment:
- [ ] [REQUIRED] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.font` class in multiple places or simply put the `font-family` property in the `body` selector and all other children elements going to inherit it by default.

- (your code)

```css
.card-work-item {
  /* ...code */
  font-family: 'Inter', 'Montserrat', sans-serif;
  /* ...code */
}

.headings-2 {
  /* ...code */
  font-family: 'Inter', 'Montserrat', sans-serif;
  /* ...code */
}
```

- suggestion

```css
.font {
  font-family: 'Inter', 'Montserrat', sans-serif;
}
```

> **Note**: Also, you can do the same thing with `display: flex` declaration.